core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ExoPlayerVideoRendererEventListener.java

```diff
@@ -1,68 +0,0 @@
-package com.novoda.noplayer.internal.exoplayer.forwarder;
-
-import android.view.Surface;
-
-import com.google.android.exoplayer2.Format;
-import com.google.android.exoplayer2.decoder.DecoderCounters;
-import com.google.android.exoplayer2.video.VideoRendererEventListener;
-
-import java.util.List;
-import java.util.concurrent.CopyOnWriteArrayList;
-
-class ExoPlayerVideoRendererEventListener implements VideoRendererEventListener {
-
-    private final List<VideoRendererEventListener> listeners = new CopyOnWriteArrayList<>();
-
-    public void add(VideoRendererEventListener listener) {
-        listeners.add(listener);
-    }
-
-    @Override
-    public void onVideoEnabled(DecoderCounters counters) {
-        for (VideoRendererEventListener listener : listeners) {
-            listener.onVideoEnabled(counters);
-        }
-    }
-
-    @Override
-    public void onVideoDecoderInitialized(String decoderName, long initializedTimestampMs, long initializationDurationMs) {
-        for (VideoRendererEventListener listener : listeners) {
-            listener.onVideoDecoderInitialized(decoderName, initializedTimestampMs, initializationDurationMs);
-        }
-    }
-
-    @Override
-    public void onVideoInputFormatChanged(Format format) {
-        for (VideoRendererEventListener listener : listeners) {
-            listener.onVideoInputFormatChanged(format);
-        }
-    }
-
-    @Override
-    public void onDroppedFrames(int count, long elapsedMs) {
-        for (VideoRendererEventListener listener : listeners) {
-            listener.onDroppedFrames(count, elapsedMs);
```

Comment:
we do not have this callback anymore?

Reply:
No we don't. The only place it is exposed by `exoplayer` is on the `VideoRendererEventListener`, which can't be attached to exoplayer directly any more.

Reply:
@Mecharyry do we still have this info in the `analytics listener`? or now we cannot know if we have dropped frames at all?

Reply:
Ah yes, `onDroppedVideoFrames` on exoplayer's AnalyticsListener is the one we want.